.appveyor.yml

```diff
@@ -1,11 +0,0 @@
-platform:
-  - x86
```

Comment:
what was this file used for?

Reply:
We currently have x86 builds, and so I thought it best to keep them. That said; I'm no longer so sure of appveyor as the right way to go about it; it might work just fine in Travis.